pkpdapp/pkpdapp/models/myokit_model_mixin.py

```diff
@@ -689,3 +722,21 @@ def _get_dosing_events(
             elif abs(start + duration - time_max) < 1e-6:
                 dosing_events[i] = (level, start, time_max - start)
     return dosing_events
+
+
+def get_project_cohorts(project):
+    dataset = project.datasets.first()
+    cohorts = []
+    if dataset is not None:
+        # TODO: create backend subject cohorts based on the
+        # frontend upload stepper
+        dataset_protocols = [
+            Protocol.objects.get(pk=p['protocol'])
+            for p in dataset.subjects.values('protocol').distinct()
+            if p['protocol'] is not None
+        ]
+        cohorts = [
+            dataset.subjects.filter(protocol=protocol)
+            for protocol in dataset_protocols
+        ]
```

Comment:
Now that subject groups are in the app, this should ignore protocols and return the groups linked to this dataset's subjects